lib/experiments-page.php

```diff
@@ -104,14 +104,14 @@ function gutenberg_initialize_experiments_settings() {
 	);
 
 	add_settings_field(
-		'gutenberg-auto-inserting-blocks',
-		__( 'Auto-inserting blocks', 'gutenberg' ),
+		'gutenberg-block-hooks',
+		__( 'Block hooks', 'gutenberg' ),
```

Comment:
An important note: remember to enable the experiment again before testing these changes. It took me a while to realize that.